apps/api/src/http/controllers/admin/fdbs/food.controller.ts

```diff
@@ -48,7 +50,10 @@ const adminFoodController = ({
       where: { id: localeId },
     });
 
+    const { code: localeCode } = await resolveLocale(localeId);
+
     const foodLocal = await adminFoodService.createFood(code, req.body);
+    await addToRedisIndexingKeyCache(localeCode, { cache });
```

Comment:
Should be keep it within the action that needs the update to happen. It seems it's pulled of out of each action into the  controllers, so will only affect potentially http layer. Though, you probably want to update the index regardless where the update goes through controller/http layer.